static_src/components/main_container.jsx

```diff
@@ -9,17 +9,23 @@ import Disclaimer from './disclaimer.jsx';
 import Header from './header.jsx';
 import Login from './login.jsx';
 import LoginStore from '../stores/login_store.js';
+import OrgStore from '../stores/org_store.js';
+import SpaceStore from '../stores/space_store.js';
 import { Nav } from './navbar.jsx';
 
-function getState() {
-  return { isLoggedIn: LoginStore.isLoggedIn() };
+function stateSetter() {
+  return {
+    currentOrgGuid: OrgStore.currentOrgGuid,
```

Comment:
In another PR, I'll be moving to this pattern of always getting the guids from the stores rather then passing the initial one down for `main`. So components will consistently get these from the stores rather then from two places (props and stores). Is that an improvement?